### PR TITLE
Added template for 0.15.0

### PR DESCRIPTION
**Pull request by azriel91**
Tested manually -- ran `cargo run -- new hello_abc && (cd hello_abc && cargo run)`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ adheres to [Semantic Versioning][sv].
 
 [sv]: http://semver.org/
 
+## 0.10.0
+
+* Added 0.15.0 template, which chooses a default graphics backend based on the developer's operating system. ([#104])
```
Comment (0x6273) [inline CHANGELOG.md:10]: This feature was also added to the 0.12.0 template. And it should probably also be added to 0.13.0 and 0.14.0, for consistency's sake.

Comment (0x6273) [reply CHANGELOG.md:10]: The link is also broken.

Comment (azriel91) [reply CHANGELOG.md:10]: ah, thanks for checking in. admittedly been rushing -- shall fix these now.

**Pull request merged**